events/source/equeue.h

```diff
@@ -26,11 +24,14 @@ extern "C" {
 #endif
 
 // Platform specific files
-#include "equeue/equeue_platform.h"
+#include "events/source/equeue_platform.h"
```

Comment:
equeue.h and equeue_platform.h should both be in internal/

Reply:
I thought we were not going for `internal` dirs but instead will only have `source`. When did this change?

Reply:
91ef36dd78